Java-9-Resources.md

```diff
@@ -180,3 +180,6 @@ Downsides/hurdles of Java 9 modularity and Project Jigsaw:
 - [java9-modularity/java9-migration-demos](https://github.com/java9-modularity/java9-migration-demos) (Sander Mak, Paul Bakker)
 - [https://github.com/java9-modularity/easytext](https://github.com/java9-modularity/java9-migration-demos) (Sander Mak, Paul Bakker), sample application from [O'Reilly's Java 9 Modularity book](http://shop.oreilly.com/product/0636920049494.do)
 - [accso/java9-jigsaw-depvis](https://github.com/accso/java9-jigsaw-depvis) (Martin Lehmann, Kristine Schaal, Rüdiger Grammes)
+- [codetojoy/WarO_Java_9_Gradle](https://github.com/codetojoy/WarO_Java_9_Gradle) (Michael Easter), example using Gradle 4.1M1
```

Comment:
Thanks for the kind words; apologies for the delay. I didn't realize that the other repos were exercises. I'll study the others and see what I can do about morphing the repos into exercises.